continuous-testing/use-cases/markmap.md

```diff
@@ -1,56 +1,115 @@
 * Continuous Testing
     * For Development APIs
-        * Unit Testing
-            * xUnit (e.g. Junit\[Java\], PyUnit\[Python\], NUnit\[C#\], CppUnit\[C++\]
-            * Tools
-                * Java
-                * Python
-                * C#
-                * C/C++
-        * Integration, Dependency and Stub Mocking
-            * Simulate dependencies
-            * Stub out coupled services for planned results
-            * Mock specific application behavior
-            * Tools
-                * Java
-                    * Mockito
-                    * EasyMock
-                    * JMock
-                * Python
-                    * 
-                * C#
-                * C/C++
-        * Code Quality Analysis
-            * Semmle
+        * For module testing (xUnit)
+            * (Java) [JUnit](https://junit.org/junit5/) [&#x1F3C1; STARTER KIT](../starter-kits/#unit-teting-with-java)
+            * (Python) [PyUnit](https://docs.python.org/3/library/unittest.html#module-unittest) [&#x1F3C1; STARTER KIT](../starter-kits/#unit-testing-with-python)
```

Comment:
If you don't have a starter kit yet, please don't include a dead-link. Leave as "STARTER KIT (TBD)" without the HREF.

Reply:
Done. But incidentally it's not a broken link (as opposed to a dead link) -- It returns to the top-level of the STARTER KIT by default when the link does not exist.